### ISSUE TITLE
Issue with Image Sequences in Remotion v4.0.232

### ISSUE THREAD

**Issue by algoholger** [remotion-dev/remotion#4580]
I encountered issues while rendering a video based on using image sequences in Remotion v4.0.232, following the [official documentation](https://www.remotion.dev/docs/assets#using-image-sequences). 

## Steps to Reproduce
1. Implement the following code as per the documentation:
   ```tsx
   import { Img, staticFile, useCurrentFrame } from "remotion";

   const MyComp: React.FC = () => {
     const frame = useCurrentFrame();

     return <Img src={staticFile(`/frame${frame}.png`)} />;
   };
   ```
2. Render out a 500-frame video (1920x1080 resolution) with different concurrency settings.

## Observed Behavior
1. **Image sequence is displayed properly in studio.**

2. **When rendering with `concurrency > 1`:**  
   Frames are encoded in the wrong order, resulting in a jumbled video output.

3. **When when rendering with `concurrency = 1`:**  
   Multiple frames are duplicated, causing a stuttering video.

## Attempted Fixes
I tried the following `Img` component props without success:
- `maxRetries`
- `pauseWhenLoading`
- `delayRenderTimeoutInMilliseconds`

## Environment
- **Remotion Version:** v4.0.232
- **Operating Systems Tested:**
  - macOS
  - Ubuntu Server

**Comment by JonnyBurger** [comment]
I tried to reproduce this by generating an image sequence with Remotion:

```ts
import {random, useCurrentFrame} from 'remotion';

export function selectColor(color: string, frame: number): number {
	return Math.floor((random(`${color}-${frame}`) * 255) % 255);
}

export const Framer: React.FC<{}> = ({}) => {
	const frame = useCurrentFrame();

	const red = selectColor('red', frame);
	const green = selectColor('green', frame);
	const blue = selectColor('blue', frame);

	return (
		<div
			style={{
				display: 'flex',
				justifyContent: 'center',
				alignItems: 'center',
				background: 'white',
				flex: 1,
				borderTop: `100px solid rgb(${red}, ${green}, ${blue})`,
				paddingBottom: 100,
			}}
		>
			<h1 style={{fontSize: 120}}>{frame}</h1>
		</div>
	);
};
```

By clicking Render and exporting an image sequence into the `public` folder

And then rendering it into a video with:

```ts
import {Img, staticFile, useCurrentFrame} from 'remotion';

export const Framer: React.FC = () => {
	const frame = useCurrentFrame();

	return (
		<div>
			<Img
				src={staticFile(
					`framer/element-${frame.toString(10).padStart(3, '0')}.png`,
				)}
			/>
		</div>
	);
};
```

This worked well for me. Created a 1920x1080 video with 30fps and 500 frames, inspected all frames separately.

Can you produce the issue using my components?
Do you have more information that could help me produce the issue?

**Comment by algoholger** [comment]
Hi, thanks for the reply and the awesome project.

I cannot reproduce the issue with your example code. I tried adding noise to the created images and using different compression levels (png/jpeg). It works flawlessly when the images are created with Remotion.

But when I create an image sequence from an iPhone movie using ffmpeg, the issue occurs as described. I tried png/jpeg and different quality levels in ffmpeg but could not get it to work. There might be something wrong with the image sequence, however eg. blender renders it fine.

If you'd like to take a look:
[image sequence 100 frames, 400 mb](https://drive.google.com/file/d/1P6NBFyN1quTf2i5Cqi1oN64pu33QkakH/view?usp=share_link)

Maybe interesting: rendering image sequences with a separate audio file runs much faster for me than using OffthreadVideo, especially with multiple videos and larger resolutions.
